tableauserverclient/server/request_factory.py

```diff
@@ -141,14 +141,16 @@ def create_req(self, schedule_item):
         schedule_element.attrib['frequency'] = interval_item._frequency
         frequency_element = ET.SubElement(schedule_element, 'frequencyDetails')
         frequency_element.attrib['start'] = str(interval_item.start_time)
-        if interval_item.end_time:
-            frequency_element.attrib['end'] = str(interval_item.end_time)
-        if interval_item.interval:
-            intervals_element = ET.SubElement(frequency_element, 'intervals')
-            for interval in interval_item.interval:
-                expression, value = interval
-                single_interval_element = ET.SubElement(intervals_element, 'interval')
-                single_interval_element.attrib[expression] = value
+        if hasattr(interval_item, 'end_time'):
+            if interval_item.end_time:
```

Comment:
you could put this on the hasattr line with an "and". I assume python does not evaluate the second expression if the first expression is false. Don't need to fix it now but seems more concise